ocaml/xapi/storage_smapiv1_wrapper.ml

```diff
@@ -422,10 +422,10 @@ functor
               | Vdi_automaton.Deactivate ->
                   Storage_migrate.pre_deactivate_hook ~dbg ~dp ~sr ~vdi ;
                   Impl.VDI.deactivate context ~dbg ~dp ~sr ~vdi ~vm ;
+                  Storage_migrate.post_deactivate_hook ~sr ~vdi ~dp ;
```

Comment:
This is SMAPIv1 only. I guess it still works for the v1 to v3 storage migration but anything migrating _from_ SMAPIv3 would need another plan.